homeassistant/components/smhi/config_flow.py

```diff
@@ -13,7 +18,7 @@
 
 
 @callback
-def smhi_locations(hass: HomeAssistant):
+def smhi_locations(hass: HomeAssistant) -> set[str]:
     """Return configurations of SMHI component."""
     return {
         (slugify(entry.data[CONF_NAME]))
```

Comment:
This looks like a single element tuple to me.

Reply:
No, there's no comma.

Python 3.8.5 (default, Jan 27 2021, 15:41:15) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.18.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: type(("test"))
Out[1]: str

Reply:
I was only aiming for fixing typing in this PR. Not clean up the rest of the code. There are more things that can be cleaned up.